### DISCUSSION TITLE
Are anonymous sign-ins persisted?

### DISCUSSION THREAD

**Discussion by MicahDoes** [supabase/supabase#31224] [Questions] [answered]
I am trying to figure out if anonymous sign-ins "stay" signed in, using something like cookies or ip. It would be pretty annoying if it were counted as a new user every time, because MAU would spike. 

How are anonymous users calculated? Thanks!

**Comment by GaryAustin1** [comment, marked as answer]
The session is stored in local storage or cookies depending on your framework.   As long as the user does not signOut or clear their browser cookies/local storage, they will stay with the same session and userid.
